KIF Tests/BackgroundTests.m

```diff
@@ -35,6 +35,9 @@ - (void)afterEach {
 }
 
 - (void)testBackgroundApp {
+#if defined(__IPHONE_14_0) // Xcode 12, UIAutomation framework not available
```

Comment:
could we just do `if (@available(iOS 14.0, *))`  ??

Reply:
done in https://github.com/openium/KIF/commit/4f286cf6f0264c1b93b5b1f860f00ca0bfb43104